pkg/vminfo/linux_syscalls.go

```diff
@@ -183,6 +184,13 @@ func linuxSyzKvmSetupCPUSupported(ctx *checkContext, call *prog.Syscall) string
 	return "unsupported arch"
 }
 
+func linuxSyzKvmVgicV3SetupSupported(ctx *checkContext, call *prog.Syscall) string {
+	if call.Name == "syz_kvm_vgic_v3_setup" && ctx.target.Arch == targets.ARM64 {
```

Comment:
linuxSyzKvmVgicV3SetupSupported is syz_kvm_vgic_v3_setup-specific, so `call.Name == "syz_kvm_vgic_v3_setup" &&` part can be removed.